arelle/ValidateXbrl.py

```diff
@@ -52,29 +64,36 @@ class ValidateXbrl:
     authority: str | None
     authParam: dict[str, Any]
     consolidated: bool
-    domainMembers: set
-    extensionImportedUrls: set
+    domainMembers: set[Any]
```

Comment:
If you (who have more experience with using the library) know what is included in these sets, please let me know!

Reply:
domain members are model concepts --> set[ModelConcept]
extensionImportedUrls is string, modelDocument.uri --> set[str]

Reply:
Thanks!